plugins.d/python.d.plugin

```diff
@@ -0,0 +1,329 @@
+#!/usr/bin/python3 -u
+
+import os
+import sys
+import time
+#import configparser
+#import importlib.machinery
+try:
+    assert sys.version_info >= (3,1)
+    import configparser
+    import importlib.machinery
```

Comment:
I understand these are the only 2 python module that force us to use python3.
Right?